utils/swift_build_support/swift_build_support/products/swiftsyntax.py

```diff
@@ -81,14 +77,22 @@ def should_build(self, host_target):
         return True
 
     def build(self, host_target):
-        self.run_swiftsyntax_build_script(target=host_target)
+        self.run_swiftsyntax_build_script(target=host_target, 
+                                          command='build')
 
     def should_test(self, host_target):
         return self.args.test_swiftsyntax
 
     def test(self, host_target):
         self.run_swiftsyntax_build_script(target=host_target,
-                                          additional_params=['--test'])
+                                          command='verify-source-code')
```

Comment:
I don’t think we need to pass `--test` here because there no longer is a `--test` argument.

Also: Could we verify source code as part of the build and only if `self.args.swiftsyntax_verify_generated_files` is `True`. I.e. have something like the following in the `build` method before running the `build` subcommand:

```python
if self.args.swiftsyntax_verify_generated_files:
  self.run_swiftsyntax_build_script(target=host_target,
                                    command='verify-source-code')
```